src/uu/fmt/src/fmt.rs

```diff
@@ -356,14 +373,14 @@ pub fn uu_app() -> Command {
             Arg::new(OPT_WIDTH)
                 .short('w')
                 .long("width")
-                .help("Fill output lines up to a maximum of WIDTH columns, default 79.")
+                .help("Fill output lines up to a maximum of WIDTH columns, default 75.")
                 .value_name("WIDTH"),
         )
         .arg(
             Arg::new(OPT_GOAL)
                 .short('g')
                 .long("goal")
-                .help("Goal width, default ~0.94*WIDTH. Must be less than WIDTH.")
+                .help("Goal width, default 0.93*WIDTH. Must be less than WIDTH.")
```

Comment:
Maybe describing this as a percentage is more intuitive: "93% of the width"? GNU does that too.